alf/metrics/metrics_test.py

```diff
@@ -166,7 +157,17 @@ def testMetric(self, metric_class, num_trajectories, expected_result,
                              metric.result())
 
     def test_average_per_step(self):
-        metric = AverageDrivingMetric(batch_size=2)
+        metric = AverageDrivingMetric(
```

Comment:
Can create this metric after `trajectories` is created, using `trajectories[0]` for metric

Reply:
Good point. Updated.